src/expression.pest

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+WHITESPACE = _{ SEPARATOR }
+
+// EOI always produces token.
+// I use eoi instead of EIO to suppress generating token.
+// See: https://github.com/pest-parser/pest/issues/304
+eoi = _{ !ANY }
+
+// You can find the grammar for actions of UpdateItem in the following link.
+// https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Expressions.UpdateExpressions.html
+set_action = { SOI ~ path ~ "=" ~ value ~ ("," ~ path ~ "=" ~ value)* ~ eoi }
+remove_action = { SOI ~ path ~ ("," ~ path)* ~ eoi }
+
+path = { attr_access ~ ("." ~ attr_access)* }
+attr_access = _{ attr_name ~ list_elem* }
+
+// Attribute name can be quoted by backticks.
+attr_name = _{ "`" ~ quoted_identifier ~ "`" | non_quoted_identifier }
+
+// You can write attribute name for a`b with a``b like MySQL.
+// https://dev.mysql.com/doc/refman/8.0/en/identifiers.html
+quoted_identifier = @{ ("``" | (!"`" ~ ANY))+ }
```

Comment:
Would you add README that we can quote attribute name surround by `?

Reply:
All right!

Reply:
I added it at https://github.com/awslabs/dynein/pull/132/commits/ad91edcc3807c792f04b54acbb0a5fd264d85cb4.